extensions/wikia/ImageReview/ImageReviewSpecialController.class.php

```diff
@@ -1,25 +1,72 @@
 <?php
 
-use \Wikia\Logger\WikiaLogger;
+use Wikia\Logger\WikiaLogger;
 
 class ImageReviewSpecialController extends WikiaSpecialPageController {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "ImageReviewSpecialController" has 25 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)